model/countries/IN/IN-model.yaml

```diff
@@ -1,61 +1,49 @@
-extra-definitions:
-  street-location: # re-ordered the fields and added the in-building-location
-  - in-building-location
-  - building
-  - street
-
-  street-address-alternative-1:
-  - in-building-location
-  - building
-  - street
-  - landmark
-  - locality2
-
-  street-address-alternative-2:
+cut-off-children:
   - street-location
-  - landmark
-  - locality2
 
-  street-address-alternative-3:
-  - in-building-location
-  - building
-  - street-and-locality2
-  - landmark
+cut-off-tokens:
+  - admin-area2
+  - admin-area3
+  - admin-area4
+  - locality3
+  - locality4
 
-  street-and-locality2:
-  - street
-  - locality2
+extra-definitions:
+  # Re-definition to remove the locality2 from the address
+  address:
+    - street-address
+    - street-address-alternative-1
+    - locality1
+    - admin-area1
+    - postal-code
+    - country
+    - country-name
 
-  street-address-alternative-4:
-  - in-building-location
-  - building
-  - street-and-landmark
+  street-location-and-locality2:
+  - street-location
   - locality2
 
-  street-and-landmark:
-  - street
+  street-address-alternative-1:
+  - street-location-and-locality2
   - landmark
 
-  street-address-alternative-5:
-  - street-location-and-landmark
-  - locality2
-
   street-location-and-landmark:
   - street-location
   - landmark
 
-  street-address-alternative-6:
-  - street-location-and-locality2
+  street-address-alternative-2:
+  - street-location-and-landmark
+  - locality2
+
+  landmark-and-locality2:
   - landmark
+  - locality2
 
-  street-location-and-locality2:
+  street-address-alternative-3:
   - street-location
-  - locality2
+  - landmark-and-locality2
 
 append-after:
   street-address-alternative-1:
   - street-address-alternative-2
```

Comment:
Do we have any intention to use street-address-alternative-2 and street-address-alternative-3? Alternatives other than 1 are being ignored so far.

Also, are we planning on introducing the concept of synthesized nodes here as well? I would imagine something like:

synthesized-nodes:
-   landmark-and-locality2
-   street-location-and-landmark